script-library/container-features/src/features.json

```
@@ -65,9 +65,9 @@
 			"options": {
 				"version": {
 					"type": "string",
-					"enum": ["latest"],
+					"proposals": ["latest"],
```

Same suggestion as above.
```suggestion
					"proposals": ["latest", "20.10"],
```